cns/restserver/api.go

```diff
@@ -17,10 +16,9 @@ import (
 	"github.com/Azure/azure-container-networking/cns"
 	"github.com/Azure/azure-container-networking/cns/hnsclient"
 	"github.com/Azure/azure-container-networking/cns/logger"
-	"github.com/Azure/azure-container-networking/cns/nmagent"
 	"github.com/Azure/azure-container-networking/cns/types"
 	"github.com/Azure/azure-container-networking/cns/wireserver"
-	"github.com/Azure/azure-container-networking/common"
+	nma "github.com/Azure/azure-container-networking/nmagent"
```

Comment:
package is used as `nmagent` in other places, any reason to rename it in some places and not on others? personally, I think renaming packages should be done as little as possible. if it's widely understood that `nma` refers to nmagent, then the package should just be `nma`.

Reply:
Any instance of this is an oversight from when there was the old and new packages both named `nmagent`. I agree, but I won't fix this in this PR because it is blocking other feature work. I'll add this to remediation items for a future PR.